Automattic/woocommerce-payments
# Remove payout timing notice and update the help tooltip.

Fixes https://github.com/Automattic/woocommerce-payments/issues/7900

#### Changes proposed in this Pull Request









Note: the changes from this PR looks different from what the original issue https://github.com/Automattic/woocommerce-payments/issues/7900 explains because there's some discussions there. If there's further discussion, I'll update this PR accordingly.

On the Payments overview page:
- Remove the payout timing notice.

![386996435-9d456c02-18cc-4284-b24b-9c0b0686cabf](https://github.com/user-attachments/assets/851f0239-4737-4e46-9ea2-dcdeae5cfc66)

- Update the help tooltip.

![389945513-1297c974-a633-45f1-a1a2-4ce377efecbd](https://github.com/user-attachments/assets/7f4ed2d5-d500-4096-8671-b549acba6f20)


#### Testing instructions





* Go to Payments overview page.
* Make sure payout notice is not there anymore.
* Make sure help tooltip is updated according to the discussion happening in https://github.com/Automattic/woocommerce-payments/issues/7900.

-------------------


**Post merge**
